app/models/user.js

```diff
@@ -33,12 +37,27 @@ module.exports = (sequelize, DataTypes) => {
         type: DataTypes.BOOLEAN,
         field: 'is_admin',
         defaultValue: false
+      },
+      points: {
+        type: DataTypes.INTEGER,
+        field: 'points',
```

Comment:
There is no need to define the `field` name if it is the same than the defined as object name

Reply:
ok, got it